freqtrade/rpc/telegram.py

```diff
@@ -170,14 +170,15 @@ def _init_keyboard(self) -> None:
         self._keyboard: list[list[str | KeyboardButton]] = [
             ["/daily", "/profit", "/balance"],
             ["/status", "/status table", "/performance"],
-            ["/count", "/start", "/stop", "/help"],
+            ["/start", "/pause", "/stop", "/help"],
```

Comment:
i don't see any reason to change the default keyboard at this point.

Reply:
I think it could be relevant to add the **pause** button in the third row with **start** and **stop**. Following this idea, count can be replaced.
I don't know how often the count button is used but as it does not provide much information can we combine the output with status table? Of course this could be done in another PR.

Reply:
if that's the argument, then the whole 3rd row is pointless, as i know near no-one that uses `/start` and `/stop` on a regular basis.

Reply:
Well as this is a customizable table, users can change that if they want, so I'll not try to guess what they need.

It just seem logic to me to have pause with start and stop, as on a media player or a tv remote 😅

Reply:
i'd leave it as is for now - this an be changed at a later point, IF we see that people actually use this :)

Reply:
Let's do this, so I revert that 👍🏽